guides/keras_cv/classification_with_keras_cv.py

```diff
@@ -0,0 +1,688 @@
+soon!
+
+Next, let's look into `MixUp()`.
+Unfortunately, while `MixUp()` has been empirically shown to *substantially*
+improve both the robustness and the generalization of the trained model,
+it is not well-understood why such improvement occurs.
+A little alchemy never hurt anyone!
+
+![](https://i.imgur.com/d4ZZYvW.png)
+
+`MixUp()` works by sampling two images from a batch, then proceeding to
+literally blend together their pixel intensities as well as their classification
+labels.
+
+Let's see it in action:
+"""
+mix_up = keras_cv.layers.MixUp()
+# MixUp needs to modify both images and labels
+inputs = {"images": image_batch, "labels": label_batch}
+
+keras_cv.visualization.plot_image_gallery(
+    mix_up(inputs)["images"],
+    rows=3,
+    cols=3,
+    value_range=(0, 255),
+    show=True,
+)
+"""
+If you look closely, you'll see that the images have been blended together.
+
+Instead of applying `CutMix()` and `MixUp()` to every image, we instead pick
+one or the other to apply to each batch.
+This can be expressed using `keras_cv.layers.RandomChoice()`
+"""
+
+cut_mix_or_mix_up = keras_cv.layers.RandomChoice([cut_mix, mix_up], batchwise=True)
+augmenters += [cut_mix_or_mix_up]
+
+"""
+Applying it to your training pipeline is easy:
+"""
+
+augmenter = keras.Sequential(augmenters)
+train_ds = train_ds.map(augmenter, num_parallel_calls=tf.data.AUTOTUNE)
+
+image_batch = next(iter(train_ds.take(1)))["images"]
+keras_cv.visualization.plot_image_gallery(
+    image_batch,
+    rows=3,
+    cols=3,
+    value_range=(0, 255),
+    show=True,
+)
+
+"""
+We also need to resize our evaluation set, but luckily that's trivial:
+"""
+inference_resizing = keras_cv.layers.Resizing(
+    IMAGE_SIZE[0], IMAGE_SIZE[1], crop_to_aspect_ratio=True
+)
+eval_ds = eval_ds.map(inference_resizing, num_parallel_calls=tf.data.AUTOTUNE)
+
+inference_resizing = keras_cv.layers.Resizing(
+    IMAGE_SIZE[0], IMAGE_SIZE[1], crop_to_aspect_ratio=True
+)
+eval_ds = eval_ds.map(inference_resizing, num_parallel_calls=tf.data.AUTOTUNE)
+
+image_batch = next(iter(eval_ds.take(1)))["images"]
+keras_cv.visualization.plot_image_gallery(
+    image_batch,
+    rows=3,
+    cols=3,
+    value_range=(0, 255),
+    show=True,
+)
+keras_cv.visualization.plot_image_gallery(
+    image_batch,
+    rows=3,
+    cols=3,
+    value_range=(0, 255),
+    show=True,
+)
+
+"""
+Finally, lets unpackage our datasets and prepare to pass them to the `model.fit()`
+call, which accepts a tuple of `(images, labels)`.
+"""
+
+
+def unpackage_dict(inputs):
+    return inputs["images"], inputs["labels"]
+
+
+train_ds = train_ds.map(unpackage_dict, num_parallel_calls=tf.data.AUTOTUNE)
+eval_ds = eval_ds.map(unpackage_dict, num_parallel_calls=tf.data.AUTOTUNE)
+
+"""
+Cool!  Data augmentation is by far the hardest piece of training a classifier
+in the modern era.
+Congratulations on making it this far!
+
+## Optimizer Tuning
+
+To achieve optimal performance, we must implement a Warm up Cosinde decay
```

Comment:
we need to use a learning rate schedule instead of a single learning rate. While we won't go into detail on the Cosine decay with warmup schedule used here...